packages/kv/src/index.ts

```diff
@@ -82,6 +82,18 @@ export function createClient(config: RedisConfigNodejs): VercelKV {
 
 // eslint-disable-next-line import/no-default-export
 export default new Proxy(
+  {},
+  {
+    // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+    get() {
+      throw new Error(
+        '"The default export has been moved to a named export, change to import { kv }"',
+      );
+    },
```

Comment:
```suggestion
    get(target, prop, receiver) {
      if (prop === 'then' || prop === 'parse') return Reflect.get(target, prop, receiver);
      throw new Error(
        '"The default export has been moved to a named export, change to import { kv }"',
      );
    },
```

Since this is going to be removed for V1, we could do this to prevent it from always throwing in Vite. It's the disgusting hacky solution we didn't want to implement for the default export, but since it'll go away at 1.0, I'd be comfortable with it.

Reply:
Not positive this exact thing would work (would have to test with Vite), but something like it should.

Reply:
Why not add a `console.warn` to the default export, saying that
```
'"The default export will be moved to a named export after the beta, change to import { kv }"',
``` 
without throwing any error?

This way the npm version is a real `minor` (no breaking changes)

To sum up:
- `0.2.0` adds the named export, fixes Vite using the hacky solution, warns every user about the upcoming breaking change
- `1.0.0` removes the `default` export as soon as we go GA

Reply:
I was able to reproduce the `Vite` issue in a jest test, I added a test case.
I changed the `throw` to a colored `console.warn`, but indeed it may not be that visible